frontend/app/routes/index.tsx

```diff
@@ -40,7 +40,7 @@ export default function RootIndex() {
     <main role="main" className="flex h-svh bg-splash-page bg-cover bg-center" property="mainContentOfPage">
       <div className="m-auto w-[300px] bg-white md:w-[400px] lg:w-[500px]">
         <div className="p-8">
-          <h1 className="sr-only">Canadian Dental Care Plan | Régime canadien de soins dentaires</h1>
+          <h1 className="sr-only"><span lang="en">Canadian Dental Care Plan </span><span lang="fr">Régime canadien de soins dentaires</span></h1>
```

Comment:
Is the space between "Plan" and `</span>` intentional?

Reply:
no ill change it up